opentelemetry-sdk/tests/context/propagation/test_b3_format.py

```diff
@@ -0,0 +1,169 @@
+# Copyright 2019, OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+import opentelemetry.trace as api_trace
+import opentelemetry.sdk.context.propagation.b3_format as b3_format
+import opentelemetry.sdk.trace as trace
+
+FORMAT = b3_format.B3Format()
+
+
+class TestB3Format(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        trace_id = trace.generate_trace_id()
+        cls.trace_id = "{:032x}".format(trace_id)
```

Comment:
I suggest using `cls.serialized_trace_id` and `cls.trace_id_decstr` as names (although it seems trace_id_internal/trace_id_decstr is unused).

Reply:
I'm not familiar with the term "decstr": what does that mean?

serialized_(trace|span)_id make a lot of sense, will rename.

Reply:
if `decstr` is confusing then probably `decimal_string` is better 😃

Reply:
ah!  got it, thanks. 

I've removed the value entirely as you mentioned it's unneeded. Thanks for explaining anyway :)